src/tables/colr.rs

```diff
@@ -91,43 +751,1153 @@ impl<'a> Table<'a> {
         let layers = Stream::new_at(data, layers_offset.to_usize())?
             .read_array16::<LayerRecord>(num_layers)?;
 
-        Some(Self {
+        let mut table = Self {
+            version: version as u8,
+            data,
             palettes,
             base_glyphs,
             layers,
-        })
+            base_glyph_paints_offset: Offset32(0), // the actual value doesn't matter
+            base_glyph_paints: LazyArray32::default(),
+            layer_paint_offsets_offset: Offset32(0),
+            layer_paint_offsets: LazyArray32::default(),
+            clip_list_offsets_offset: Offset32(0),
+            clip_list: ClipList::default(),
+            #[cfg(feature = "variable-fonts")]
+            item_variation_store: None,
+            #[cfg(feature = "variable-fonts")]
+            var_index_map: None,
+        };
+
+        if version == 0 {
+            return Some(table);
+        }
+
+        table.base_glyph_paints_offset = s.read::<Offset32>()?;
+        let layer_list_offset = s.read::<Option<Offset32>>()?;
+        let clip_list_offset = s.read::<Option<Offset32>>()?;
+        #[cfg(feature = "variable-fonts")]
+        let var_index_map_offset = s.read::<Option<Offset32>>()?;
+        #[cfg(feature = "variable-fonts")]
+        let item_variation_offset = s.read::<Option<Offset32>>()?;
+
+        {
+            let mut s = Stream::new_at(data, table.base_glyph_paints_offset.to_usize())?;
+            let count = s.read::<u32>()?;
+            table.base_glyph_paints = s.read_array32::<BaseGlyphPaintRecord>(count)?;
+        }
+
+        if let Some(offset) = layer_list_offset {
+            table.layer_paint_offsets_offset = offset;
+            let mut s = Stream::new_at(data, offset.to_usize())?;
+            let count = s.read::<u32>()?;
+            table.layer_paint_offsets = s.read_array32::<Offset32>(count)?;
+        }
+
+        if let Some(offset) = clip_list_offset {
+            table.clip_list_offsets_offset = offset;
+            let clip_data = data.get(offset.to_usize()..)?;
+            let mut s = Stream::new(clip_data);
+            // Format
+            s.read::<u8>()?;
+            let count = s.read::<u32>()?;
+            table.clip_list = ClipList {
+                data: clip_data,
+                records: s.read_array32::<ClipRecord>(count)?,
+            };
+        }
+
+        #[cfg(feature = "variable-fonts")]
```

Comment:
Have you tried parsing a variable colrv1 font without this feature? Just to make sure.

Reply:
Yep, just tried it with `font2svg` and then the variable parts are missing.